docs/en/token_endpoint.rst

```diff
@@ -71,22 +72,22 @@ The authentication method of the RP by the Token Endpoint is the private_key_jwt
      - **Description**
      - **Supported by**
    * - **client_id**
-     - URI that uniquely identifies the RP. 
+     - See `OpenID.Registration`_. It MUST contain an HTTPS URL that uniquely identifies the RP.
      - |spid-icon| |cieid-icon|
    * - **client_assertion**
      - JWT signed with the Relying Party's private key containing the following parameters:
 	 
-	 **iss**: RP identifier registered by the OP, that uniquely identifies the Federation's Entity in the Uniform Resource Locator (URL) format. Corresponds to the client_id, used in the authentication request.
+	 **iss**: This MUST contain the *client_id*.
 	 
-	 **sub**: equal to the parameter **iss**.
+	 **sub**: This MUST contain the *iss*.
 	 
 	 **aud**: URL of the OP Token Endpoint.
 	 
 	 **iat**: UNIX Timestamp with the time of the JWT generation, coded as NumericDate as indicated in :rfc:`7519`. 
 	 
 	 **exp**: UNIX Timestamp with the expiry time of the JWT, coded as NumericDate as indicated in :rfc:`7519`. 
 	 
-	 **jti**: Unique Identifier uuid4 for this authentication request, generated by the client.
+	 **jti**: Unique Identifier uuid4 for this authentication request, generated by the client. It MUST be in *uuid4* format
```

Comment:
not sure that it MUST be uuid4, can we suggest this or is it mandatory as SPID and CIE LGs?

Reply:
The format of jti is not specified for SPID. I think it's can be suggested only.

Reply:
Done here
https://github.com/italia/spid-cie-oidc-docs/pull/51/commits/f2ff72a6cfa4e8145b43727bf4f44abc8bc23a99